apis/placement/v1alpha1/clusterresourceplacement_types.go

```diff
@@ -3,7 +3,7 @@ Copyright (c) Microsoft Corporation.
 Licensed under the MIT license.
 */
 
-package v1
+package v1alpha1
```

Comment:
it's better to mark this as the storage version or remove that tag for all the objects. I only moved the new CRDs to storage version when I made the first change to avoid breaking existing tests

Reply:
updated the storage flag from the api and regenerated the manifests.
All the yaml files set "storage: true" by default.